src/lib/libfs.js

```diff
@@ -1943,7 +1931,7 @@ for (let key in LibraryFS.$FS) {
   if (LibraryFS[alias]) {
     continue;
   }
-  LibraryFS[alias] = `FS.${key}`;
+  LibraryFS[alias] = `(...args) => FS.${key}(...args)`;
```

Comment:
Is the forwarding function needed?  (Rather than just the string?)

Reply:
Yeah, depending on the ordering of the output some of these functions are output before `FS` is defined.

Reply:
Hmm.. I wonder how that is possible give the `__deps` entry?  IIRC deps should always come first.

I guess its not a big deal.

Reply:
I was wondering about that too. It's when I remove `FS_createDataFile` `other.test_exported_runtime_methods` fails.